.github/workflows/conformance-aks.yaml

```diff
@@ -260,7 +254,7 @@ jobs:
 
       - name: Enable Relay
         run: |
-          cilium hubble enable ${{ steps.vars.outputs.hubble_enable_defaults }}
+          cilium hubble enable
```

Comment:
should we remove this step and add enabled flag with helm-set ?

Reply:
that works too, i don't have a strong preference here 🧘

Reply:
let's do this as a follow up if we want to remove this step